doc/Makefile

```diff
@@ -40,9 +40,10 @@ help:
 .PHONY: clean
 
 clean:
-	rm -rf $(BUILDDIR)/*
+	rm -rf $(BUILDDIR)/
```

Comment:
okay, as long as you don't break `make html`, fine with me. Did you try running `make html` after `make clean` to see if it still works?